source/modules/MEI.cmn.xml

```diff
@@ -2383,7 +2383,7 @@
         that is, for repetition of the same pitch. In the case of irrational durations, such as such
         as two quarter notes in the time of five 8th notes in a measure of 5/8 time, decimal values
         may be used in the <att>dur.ges</att> attribute. For example, the <att>dur.ges</att>
-        attribute would take the value "2.5" if the <att>midi.div</att> attribute's value was "1".
+        attribute would take the value <val>2.5</val> if the <att>midi.div</att> attribute’s value was <val>1</val>.
         The <att>num</att> and <att>numbase</att> attributes may be used for explicit labelling of a
         tuplet, such as, '3' with an 8th-note triplet, '3:2' over a quarter-note triplet, etc. The
```

Comment:
Sorry, just another observation: If the `2.5` and `1` above are embraced by `val` tags, `3` and `3:2` should be tagged here, too, shouldn't they?

Reply:
`3:2` is not an allowed value in these attributes, so I think both, number and ratio, are supposed to show what "meaning" can be expressed with the two attributes.